### PR TITLE
Task_2: solving math example

### PR DESCRIPTION
**Pull request by SachenkoTanya**

### CODE REVIEW

**Comment by vloooo** [comment]
please change the PR's title. I mentioned the rules several times

**Comment by vloooo** [comment]
branch naming too

File: Task2.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#Multiparadigm programming languages,  Task2
+#Sachenko Tanya,  IKM-221a
+
+print('Multiparadigm programming languages,  Task2')
+print('Sachenko Tanya,  IKM-221a', end='\n\n')
+
+x = float(input('Input  x  please: '))
+y = float(input('Input  y  please: '))
+z = float(input('Input  z  please: '))
```
Comment (vloooo) [inline Task2.py]: use ''Input  {}  please: ' as template.
try not to repeat the same strings and pieces of code

File: Task2.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#Multiparadigm programming languages,  Task2
+#Sachenko Tanya,  IKM-221a
+
+print('Multiparadigm programming languages,  Task2')
+print('Sachenko Tanya,  IKM-221a', end='\n\n')
+
+x = float(input('Input  x  please: '))
```
Comment (vloooo) [inline Task2.py]: In theory, user can type 'Sachenko Tanya'. or any other sequence of alphabetical characters
How is your program going to deal with such type of input?

File: Task2.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+y = float(input('Input  y  please: '))
+z = float(input('Input  z  please: '))
+
+if x == -11.3 or z == 11.3:
+    print('Error!!!')
```
Comment (vloooo) [inline Task2.py]: I would propose you raise an error.
https://www.w3schools.com/python/gloss_python_raise.asp

File: Task2.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+else:
+    print((x / (x + 11.3)) + (y / (z - 11.3)))
+
```
Comment (vloooo) [inline Task2.py:16]: do you have only 1 blank line at the end of the file?

File: Task2.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#Multiparadigm programming languages,  Task2
+#Sachenko Tanya,  IKM-221a
```
Comment (vloooo) [inline Task2.py]: take as a rule to add a space between **#** and a comment string

File: Task2.py (hunk 4 above)
Comment (SachenkoTanya) [reply Task2.py:16]: Yes.How many empty lines should there be?

File: Task2.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+print('Sachenko Tanya,  IKM-221a', end='\n\n')
+
+
+def get_float_input(message):
+    while True:
+        try:
+            value = float(input(message))
+            return value
+        except ValueError:
+            print("Please enter a valid number!")
```
Comment (vloooo) [inline Task2.py]: double quotes

File: Task2.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# Multiparadigm programming languages,  Task2
+# Sachenko Tanya,  IKM-221a
+
+print('Multiparadigm programming languages,  Task2')
+print('Sachenko Tanya,  IKM-221a', end='\n\n')
+
+
+def get_float_input(message):
```
Comment (vloooo) [inline Task2.py:8]: like such approach

File: Task2.py [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+
+INPUT_MESSAGE = 'Input  {}  please: '
+x = get_float_input(INPUT_MESSAGE.format('x'))
+y = get_float_input(INPUT_MESSAGE.format('y'))
+z = get_float_input(INPUT_MESSAGE.format('z'))
+
+if x == -11.3 or z == 11.3:
+    raise Exception('Error!!!')
+else:
```
Comment (vloooo) [inline Task2.py]: you don't need this **else**

**Pull request merged**